docs/config.md

```diff
@@ -138,6 +138,11 @@ The explanation and default value of each configuration item are as follows:
       # runtime_type is the runtime type to use in containerd e.g. io.containerd.runtime.v1.linux
       runtime_type = "io.containerd.runc.v1"
 
+      # pod_annotations is list of pod annotations passed to OCI annotations
```

Comment:
Are these annotations passed to the pod sandbox, or containers, or both the pod sandbox and all containers?

Let's clarify that.

Reply:
Done